README.md

```diff
@@ -44,3 +44,296 @@ Please read our
 ## License
 
 GNU GPL version 2 or later
+
+## Contributors
+
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+
+All contributions to this project are gratefully acknowledged using the [`allcontributors` package](https://github.com/ropenscilabs/allcontributors) following the [all-contributors](https://allcontributors.org) specification. Contributions of any kind are welcome!
+
+<table>
+
+<tr>
+<td align="center">
+<a href="https://github.com/gaborcsardi">
+<img src="https://avatars.githubusercontent.com/u/660288?v=4" width="100px;" alt=""/>
+</a><br>
+<a href="https://github.com/maelle/rigraph/commits?author=gaborcsardi">gaborcsardi</a>
```

Comment:
probably `allcontributors::add_contributors()` would need to be run again from upstream not my fork :-)